notification/app/notification/services/frontend/FrontendAlerts.scala

```diff
@@ -0,0 +1,46 @@
+package notification.services.frontend
+
+import models.{BreakingNewsNotification, NotificationReport}
+import play.api.Logger
+import play.api.libs.json.Json
+import play.api.libs.ws.WSClient
+import play.mvc.Http.Status.CREATED
+import tracking.{TrackingError, TrackerResult, TrackingObserver}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scalaz.\/
+import scalaz.syntax.either._
+
+class FrontendAlerts(config: FrontendAlertsConfig, wsClient: WSClient) extends TrackingObserver {
```

Comment:
I don't think the fact the notification goes to a mobile device is reflected in the contract of NotificationSender, in fact we are notifying front-end and I still think we should treat it as something we notify more than something we report to